.github/workflows/test.yml

```diff
@@ -0,0 +1,70 @@
+name: "Run tests"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  test:
+    name: "Run unit tests"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php:
+          - '5.4'
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0'
+
+        include:
+          - php: '8.0'
```

Comment:
Until PHPCS has full PHP 8.0 support, it may be safer to run the sniffs on PHP 7.4.

Reply:
Also: might be an idea to have a separate workflow or job for PHPCS instead of combining this with the tests.
PHPCS does not need the matrix after all.

Reply:
I've split the steps and code sniffing now runs separately, but it results in a small bit of repeated code.

Shouldn't be a real issue, especially if later on it's decided to add more static analysis.